src/styles/fvtt/dialog.less

```diff
@@ -31,15 +35,15 @@
         justify-content: center;
         height: max-content;
         border-width: 1px;
+        border-radius: 0;
```

Comment:
Looks like this removed the border radius from the preroll dialog action buttons, and that feels a bit weird. What do you think?

![CleanShot 2022-12-03 at 06 04 23](https://user-images.githubusercontent.com/39902/205444763-56b46401-f7fc-4248-9c16-4ca413073d7f.jpg)